src/Sarif.Driver/DriverExtensionMethods.cs

```diff
@@ -169,6 +169,10 @@ public static bool ValidateOutputOptions(this AnalyzeOptionsBase options, IAnaly
             // baseline process now depends on output file
             valid &= !(string.IsNullOrWhiteSpace(options.OutputFilePath) && !string.IsNullOrWhiteSpace(options.BaselineSarifFile));
 
+            // post SARIF process now depends on output file
+            valid &= !(string.IsNullOrWhiteSpace(options.OutputFilePath) && !string.IsNullOrWhiteSpace(options.PostUri));
+            valid &= !(!string.IsNullOrWhiteSpace(options.PostUri) && !Uri.IsWellFormedUriString(options.PostUri, UriKind.Absolute));
```

Comment:
>string [](http://example.com/codeflow?start=24&length=6)

Please rewrite this logic to make it clearer.

Boolean patterns like '(!' are a clue this code isn't obvious.

Just use readable if {} conditions. This is a case where being clever and putting everything on a single line is working against maintainability.

Reply:
sure! :)